src/components/berths/Berth/types.ts

```diff
@@ -22,7 +22,7 @@ export interface Berth {
   maximumWidth: number;
   maximumLength: number;
   availabilityLevel: {
-    identifier: string;
+    id: string;
```

Comment:
`availabilityLevel` and `boatType` have integer IDs

Reply:
BUT, technically speaking you will send it as string, probably. so this is okay 👍

Reply:
ok, sorry for going back and forth with this, BUT, mutation API will only accept integers, when you will try to pass `boatType`'s ID, so...

Reply:
I think `ID`s will always be serialized as `String`s.

Reply:
So there's no need to change them on the backend.